tutorials: added pov generation theory

vitalych opened the pull request:
Signed-off-by: Vitaly Chipounov <vitaly@cyberhaven.io>

adrianherrera commented on src/Tutorials/pov.rst:
Should this say something like `by tweaking the receive syscall in the Decree kernel (a reduced Linux kernel created specifically for the CGC)` instead?

vitalych replied:
I should probably instead mention static binaries and not refer to CGC here.

adrianherrera commented on src/Tutorials/pov.rst:
The "special machine instruction" may be true for S2E, but I don't think it's necessarily true for other symbolic execution engines. Since this paragraph seems to be discussing symbolic execution engines in general, maybe this should be reworded.

vitalych replied:
Good point

adrianherrera commented on src/Tutorials/pov.rst:
Wrap "a" in double back-ticks for consistency.

vitalych replied:
ok

adrianherrera commented on src/Tutorials/pov.rst:
Another \`\`a\`\`.

adrianherrera commented on src/Tutorials/pov.rst:
Why is this formatted weirdly?

vitalych replied:
Yeah, I wanted to add a Note paragraph. RST supports ``.. note::`` but unfortunately, Github doesn't render it properly. Just identing the paragraph creates a vertical bar next to it, so it looks better. Maybe you have other ideas on how to fix it.

adrianherrera commented on src/Tutorials/pov.rst:
Why is this paragraph indented and separated by two lines?

adrianherrera replied:
Hmm and the next paragraph too

vitalych replied:
Because I couldn't find better ways of highlighting this as a note. The best I could come up with is this, which is done by an extra indent.

![image](https://user-images.githubusercontent.com/2332715/36175463-cffa986a-110f-11e8-978d-a984f21e6c81.png)

adrianherrera commented:
Also there are a few paragraphs that are weirdly indented and separated by >1 empty lines. Maybe there's a reason for this, but it doesn't produce any different reStructuredText

vitalych commented:
@adrianherrera thanks for the thorough review! I addressed the issues except for the indent stuff, where I don't see a better way of making it look nice on github.
